src/tests/specs/quickPreview.spec.ts

```diff
@@ -39,4 +39,15 @@ describe('planner quick preview tests', () => {
     await planner.quickPreview.addLink(c.linkType, c.workItemTitle4);
     expect(await planner.quickPreview.hasLinkedItem(c.workItemTitle4)).toBeTruthy();
   });
+
+  it('description box should not be open for wis',async () =>{
+    await planner.workItemList.clickWorkItem(c.workItemTitle1);
+    await planner.quickPreview.openDescriptionBox();
+    expect(await planner.quickPreview.descriptionSaveButton.isDisplayed()).toBeTruthy();
+  
+    // Open another WI(Note: the description box is still in edit mode)
+    await planner.workItemList.clickWorkItem(c.workItemTitle2);
+    // The description box should not be in edit mode 
+    expect(await planner.quickPreview.descriptionSaveButton.isDisplayed()).toBeFalsy();
```

Comment:
add this isDisplayed() in try catch block otherwise it will fail with NoSuchElementException